aws/routers.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_elb" "router" {
+  name = "${var.env}-cf-router-elb"
+  subnets = ["${aws_subnet.infra.*.id}"]
+  security_groups = [
+    "${aws_security_group.web.id}",
+  ]
+
+  health_check {
+    target = "TCP:443"
+    interval = "${var.health_check_interval}"
+    timeout = "${var.health_check_timeout}"
+    healthy_threshold = "${var.health_check_healthy}"
+    unhealthy_threshold = "${var.health_check_unhealthy}"
+  }
+  listener {
+    instance_port = 80
+    instance_protocol = "http"
+    lb_port = 80
+    lb_protocol = "http"
+  }
+  listener {
+    instance_port = 443
+    instance_protocol = "tcp"
+    lb_port = 443
+    lb_protocol = "tcp"
+  }
+  listener {
+    instance_port = 443
+    instance_protocol = "tcp"
+    lb_port = 4443
+    lb_protocol = "tcp"
+  }
```

Comment:
Yes, the CF template uses the port 4443 for the loggregator. The reason is that loggregator uses websockets and this way is easier to use different configuration for it. 

We could change it to force loggregator to use 443, but we tried to keep the changes to a minimum.